src/content/docs/en/tutorial/3-components/4.mdx

```diff
@@ -46,86 +46,86 @@ Create a `<Hamburger />` component to open and close your mobile menu.
 
 3. Place this new `<Hamburger />` component just before your `<Navigation />` component in `Header.astro`. 
 
-<details>
-<summary>Show me the code!</summary>
+    <details>
+    <summary>Show me the code!</summary>
 
-```astro title="src/components/Header.astro" ins={2,7}
----
-import Hamburger from './Hamburger.astro';
-import Navigation from './Navigation.astro';
----
-<header>
-  <nav>
-    <Hamburger />
-    <Navigation />
-  </nav>
-</header>
-```
-</details>
+    ```astro title="src/components/Header.astro" ins={2,7}
+    ---
+    import Hamburger from './Hamburger.astro';
+    import Navigation from './Navigation.astro';
+    ---
+    <header>
+      <nav>
+        <Hamburger />
+        <Navigation />
+      </nav>
+    </header>
+    ```
+    </details>
 
 4. Add the following styles for your Hamburger component:
 
-  ```css title="src/styles/global.css" ins={2-13, 56-58}
-  /* nav styles */
-  .hamburger {
-    padding-right: 20px;
-    cursor: pointer;
-  }
-
-  .hamburger .line {
-    display: block;
-    width: 40px;
-    height: 5px;
-    margin-bottom: 10px;
-    background-color: #ff9776;
-  }
-
-  .nav-links {
-    width: 100%;
-    top: 5rem;
-    left: 48px;
-    background-color: #ff9776;
-    display: none;
-    margin: 0;
-  }
-
-  .nav-links a {
-    display: block;
-    text-align: center;
-    padding: 10px 0;
-    text-decoration: none;
-    font-size: 1.2rem;
-    font-weight: bold;
-    text-transform: uppercase;
-  }
-
-  .nav-links a:hover, a:focus {
-    background-color: #ff9776;
-  }
-
-  .expanded {
-    display: unset;
-  }
-
-  @media screen and (min-width: 636px) {
-    .nav-links {
-      margin-left: 5em;
+    ```css title="src/styles/global.css" ins={2-13, 56-58}
+    /* nav styles */
+    .hamburger {
+      padding-right: 20px;
+      cursor: pointer;
+    }
+
+    .hamburger .line {
       display: block;
-      position: static;
-      width: auto;
-      background: none;
+      width: 40px;
+      height: 5px;
+      margin-bottom: 10px;
+      background-color: #ff9776;
+    }
+
+    .nav-links {
+      width: 100%;
+      top: 5rem;
+      left: 48px;
+      background-color: #ff9776;
+      display: none;
+      margin: 0;
     }
 
     .nav-links a {
-      display: inline-block;
-      padding: 15px 20px;
+      display: block;
+      text-align: center;
+      padding: 10px 0;
+      text-decoration: none;
+      font-size: 1.2rem;
+      font-weight: bold;
+      text-transform: uppercase;
     }
 
-    .hamburger {
-      display: none;
+    .nav-links a:hover, a:focus {
```

Comment:
This is another reason not to include several files in one PR. It took me several minutes of closely inspecting here to figure out why this is showing as a change, and whether or not something had actually changed by mistake.

Reply:
Yeah, this really diffed quite strangely... sorry about that.